onboarding/src/Components/UserEdit/EmployeeEditPage.js

```diff
@@ -0,0 +1,48 @@
+import React, { useEffect, useState } from "react";
+import UserEditForm from "./UserEditForm";
+import PageAddressBar from "../PageAddressBar"
+import { useLocation, useParams } from "react-router-dom";
+import { getUserById } from "../hooks/Users";
+
+function EmployeeEditPage() {
+    document.title = "Onboarding: profil pracownika";
+    const location = useLocation();
+    const [singleUser, setSingleUser] = useState({});
+    const { employee_id } = useParams();
+
+    let user;
+    if(location.state) {
+      user = location.state.user;
+    } else {
+      const { data } = getUserById(employee_id);
+      user = data;
+    };
+    
+    useEffect(() => {
+        location.state && user && setSingleUser(user)
+        !location.state && user && setSingleUser({
+            ...user,
+            tel: user.phone_number || "",
+            position: user.job_position || "", 
+            department: user.team || "", 
+        });
```

Comment:
Sorry for repetition ... but again:
"Less slowing down branching" :wink:  :-)
```
if(user){
        location.state ? setSingleUser(user) : setSingleUser({
            ...user,
            tel: user.phone_number || "",
            position: user.job_position || "", 
            department: user.team || "", 
        });
}
```
It **was** also in `onboarding/src/Components/EmployeeProfile/EmployeeProfilePage.js`